types/tektelic/edoctorV1/default.schema.json

```
@@ -0,0 +1,62 @@
+{
+  "$id": "https://akenza.io/tektelic/edoctorV1/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
```

```suggestion
  "title": "Tektelic E-Doctor V1 - Default",
```

Nitpick, but not sure if we should be a more specific in the title?